interoperability-layer-openhim/docker-compose.yml

```diff
@@ -2,7 +2,7 @@ version: '3.9'
 
 services:
   openhim-core:
-    image: jembi/openhim-core:7
+    image: jembi/openhim-core:7.2.1
```

Comment:
let's remove this change and point this PR as a dependency of this https://github.com/jembi/platform/pull/221 because the other PR contains the configs and other stuff. I will update the version of openhim-core once we release a new version